pkgs/servers/matrix-synapse/matrix-appservice-irc/node-composition.nix

```diff
@@ -0,0 +1,17 @@
+# This file has been generated by node2nix 1.8.0. Do not edit!
```

Comment:
Please use node2nix 1.9.0 which has some important changes required to fix eval.